packages/@navikt/boxed-list-with-selection/src/SelectionItem.tsx

```diff
@@ -0,0 +1,41 @@
+import bem from '@navikt/nap-bem-utils';
+import { Normaltekst } from 'nav-frontend-typografi';
+import * as React from 'react';
+import { ListItemProps } from './BoxedListWithSelection';
+import './list';
+
+interface SelectionItemProps extends ListItemProps {
+    index: number;
+}
+
+const boxedListWithSelectionItemCls = bem('boxedList__selectItem');
+
+const SelectionItem = ({ name, href, selected, callback, index }: SelectionItemProps): JSX.Element => {
```

Comment:
Sleit med navngivning her. Kom gjerne med forslag

Reply:
Er tanken her å støtte `onClick` samtidig som kan kan sende inn en `href`? Er det noe spesielt her som tilsier at vi trenger å støtte `<button>`?

Hvis ikke tenker jeg vel at dette burde være som før, og at man heller kan legge inn en `onClick` på anchor-elementet. Komponenten blir uforutsigbar og vanskelig å forstå dersom vi legger inn dynamikk på bruken av `callback` og `href`, og på hvilke elementer som rendres, og hvilke callbacks som kalles, utifra de. Typiske eksempler på spørsmål jeg tenker kan komme opp: 
- Hvorfor blir ikke onClick-callbacket mitt kalt selv om jeg har oppgitt en `href`? 
- Når får jeg en anchor? 
- Når får jeg button?

Reply:
Tanken er at enten har du onClick eller så har du href. Et anchor bør vel egentlig(?) bare brukes dersom det har en href og fører til en endring i url. En click-event i tillegg til navigeringen på et anchor føles litt rart for meg. Dersom man skal ha en onClick tenker jeg at det blir mer semantisk riktig med en button. Kan hende jeg tenker feil her. Enig i at det blir mer utforutsigbart for utvikler.

Reply:
Foreldrepenger ønsker visst å kun sende inn callback, og ikke href =/